internal/terraform/module/operation/operation.go

```diff
@@ -36,4 +36,5 @@ const (
 	OpTypeReferenceValidation
 	OpTypeTerraformValidate
 	OpTypeParseStackConfiguration
+	OpTypeLoadTerraformVersion
```

Comment:
Should we mention `Stack` in the job name here to avoid confusion? We already have `OpTypeGetTerraformVersion` to get the version from the Terraform CLI